cardano-api/gen/Gen/Cardano/Api/Typed.hs

```diff
@@ -170,8 +169,8 @@ genScriptData =
     genInteger :: Gen Integer
     genInteger = Gen.integral
                   (Range.linear
-                    (-fromIntegral (maxBound :: Word64) :: Integer)
-                    ( fromIntegral (maxBound :: Word64) :: Integer))
+                    (fromIntegral (minBound :: Word64) :: Integer)
+                    (fromIntegral (maxBound :: Word64) :: Integer))
```

Comment:
What's the reason for changing the generated value to be in the range `[-maxBound, maxBound]` to `[0, maxBound]`?

Reply:
https://input-output-rnd.slack.com/archives/C21UF2WVC/p1631459558223700?thread_ts=1631236451.216300&cid=C21UF2WVC